WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -282,7 +284,31 @@ public void onClick(View v) {
         mBtnTrashComment.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                moderateComment(CommentStatus.TRASH);
+                if (mComment.getStatusEnum() == CommentStatus.TRASH) {
```

Comment:
This would benefit from a comment stating that the dialog is necessary because trashing an already trashed comment deletes it permanently.